tests/sql.py

```diff
@@ -306,6 +309,9 @@ def test_numeric(self):
         self.assertRaises(RuntimeError, self.db.execute, "INSERT INTO foo VALUES (:1, :2)", 'bar', 'baz', 'qux')
         self.assertRaises(RuntimeError, self.db.execute, "INSERT INTO foo VALUES (:1, :2)", 'bar', baz='baz')
 
+    def test_cte(self):
+        self.assertEqual(self.db.execute("WITH foo AS ( SELECT 1 AS bar ) SELECT bar FROM foo"), [{"bar": 1}])
+
```

Comment:
Is that a duplicate test?

Reply:
MySQL < 8.0 doesn't support CTEs, so I added `test_cte` to the SQLite tests and PostgreSQL tests but not MySQL!